pebblo_saferetriever/langchain/identity-rag/pebblo_semantic_rag.py

```diff
@@ -0,0 +1,179 @@
+# Fill-in OPENAI_API_KEY in .env file in this directory before proceeding
```

Comment:
Is this file really needed under identity-rag? I think it's primary aim is semantic enforcement, we are adding identity as well within it though, but they are not dependent on each other.

Reply:
Moved to a new directory